File: Linguagem Java/Exercicios/SomaImpares.java
import java.util.Locale;
import java.util.Scanner;

public class SomaImpares {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int x,y,troca,soma;
		
		System.out.println("Digite dois numeros:");
		x = sc.nextInt();
		y = sc.nextInt();
		
		if (x > y) {
			troca = x;
			x = y;
			y = troca;			
		};
		
		soma = 0;
		for (int i=x;i<y;i++) {
			if (i % 2 != 0) {
				soma = soma + i;				
			};			
		};
		
		System.out.println("SOMA DOS IMPARES = "+soma);
		
		sc.close();
	}

}

File: Linguagem C#/Repetitiva/SomaImpares.cs
using System;

namespace SomaImpares {
    class Program {
        static void Main(string[] args) {

            int x, y, troca, soma;

            Console.WriteLine("Digite dois numeros: ");
            x = int.Parse(Console.ReadLine());
            y = int.Parse(Console.ReadLine());

            if (x > y) {
                troca = x;
                x = y;
                y = troca;
            };

            soma = 0;
            for (int i = x+1; i < y; i++) {
                if (i % 2 != 0) {
                    soma = soma + i;
                };
            };
            Console.WriteLine("SOMA DOS IMPARES = "+soma);
        }
    }
}
